codec-http2/src/main/java/io/netty/handler/codec/http2/StreamBufferingEncoder.java

```diff
@@ -89,7 +88,7 @@ public long errorCode() {
             return errorCode;
         }
 
-        public ByteString debugData() {
+        public byte[] debugData() {
```

Comment:
Do we need to call `close()` on the array ? OR should me maybe return a ByteBuf or ByteBuffer ?